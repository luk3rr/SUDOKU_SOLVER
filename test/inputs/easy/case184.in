000300009 600004700 090010032 000020400 009803000 725040000 064900800 000000300 050006004 
