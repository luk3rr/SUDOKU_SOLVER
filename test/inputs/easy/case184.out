Solving the following grid:
0 0 0 | 3 0 0 | 0 0 9 
6 0 0 | 0 0 4 | 7 0 0 
0 9 0 | 0 1 0 | 0 3 2 
------+-------+------
0 0 0 | 0 2 0 | 4 0 0 
0 0 9 | 8 0 3 | 0 0 0 
7 2 5 | 0 4 0 | 0 0 0 
------+-------+------
0 6 4 | 9 0 0 | 8 0 0 
0 0 0 | 0 0 0 | 3 0 0 
0 5 0 | 0 0 6 | 0 0 4 


Solution found :')

5 7 2 | 3 6 8 | 1 4 9 
6 1 3 | 2 9 4 | 7 5 8 
4 9 8 | 7 1 5 | 6 3 2 
------+-------+------
8 3 6 | 5 2 9 | 4 7 1 
1 4 9 | 8 7 3 | 5 2 6 
7 2 5 | 6 4 1 | 9 8 3 
------+-------+------
2 6 4 | 9 3 7 | 8 1 5 
9 8 1 | 4 5 2 | 3 6 7 
3 5 7 | 1 8 6 | 2 9 4 

Algorithm: BFS
Total time: 48 ms
Total expanded states: 7376
